beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -270,13 +281,17 @@ func (vs *Server) getPayloadHeaderFromBuilder(ctx context.Context, slot primitiv
 		if err != nil {
 			return nil, errors.Wrap(err, "could not get blob kzg commitments")
 		}
-		if len(kzgCommitments) > fieldparams.MaxBlobsPerBlock {
-			return nil, fmt.Errorf("builder returned too many kzg commitments: %d", len(kzgCommitments))
+	}
+
+	var executionRequests *enginev1.ExecutionRequests
+	if bid.Version() >= version.Electra {
+		eBid, ok := bid.(builder.BidElectra)
+		if !ok {
+			return nil, errors.New("builder returned non-electra bid")
 		}
-		for _, c := range kzgCommitments {
```

Comment:
removing this validation as we already validate when we convert the bid from json to proto